luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/ResourceNodeSupplierTest.groovy

```diff
@@ -0,0 +1,52 @@
+package com.yahoo.bard.webservice.config.luthier
+
+import com.fasterxml.jackson.databind.node.ArrayNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+import spock.lang.Specification
+
+class ResourceNodeSupplierTest extends Specification {
+    ResourceNodeSupplier defaultResourceNodeSupplier
```

Comment:
Since we're only using this in one test ("Node Existence"), we can define it in a
`given` block in that test. No reason to make it a class field.

Reply:
Changed.